ibm/resource_ibm_cis_healthcheck.go

```diff
@@ -24,6 +24,7 @@ const (
 	cisGLBHealthCheckAllowInsecure   = "allow_insecure"
 	cisGLBHealthCheckCreatedOn       = "create_on"
 	cisGLBHealthCheckModifiedOn      = "modified_on"
+	cisGLBHealthCheckHeader          = "header"
```

Comment:
header is not been included in schema..
header configurations are not been added in resource..

Reply:
I have removed this line. I will commit this feature in another PR.